mappings/net/minecraft/entity/passive/VillagerEntity.mapping

```diff
@@ -15,6 +15,7 @@ CLASS ava net/minecraft/entity/passive/VillagerEntity
 	FIELD bF levellingUp Z
 	FIELD bG lastCustomer Lavn;
 	FIELD bH buddyGolemId Ljava/util/UUID;
+	FIELD bI oneMinAfterLastGolemCheckTimestamp J
```

Comment:
I guess this is `nextGolemCheckTimestamp`?

Reply:
no. This is a trash field left by proguard; it is only set but never read. every time it is set, it is set to one minute after last golem check time if the field's value is less than that.

Reply:
It's state influences the return here, I don't think it's a trash field. ![here](https://i.imgur.com/xLoTHvJ.png)

Reply:
Hm, I guess this code is wrong. Might be a bug, the condition would be true every tick